src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -457,6 +461,13 @@ HashSet<string> ParseProfilers (string value)
 			return results;
 		}
 
+		CompressionMethod GetCompressionMethod (string fileName)
+		{
+			if (uncompressedFileExtensions.Any (x => string.Compare (x, Path.GetExtension (fileName), StringComparison.OrdinalIgnoreCase) == 0))
```

Comment:
Should we check for `uncompressedFileExtensions` being `null` here?

Reply:
nope, we check for `null` at https://github.com/xamarin/xamarin-android/pull/1827/files#diff-81b01bf217c8f8113c5cdd8ccf840eedR235

Reply:
But if `UncompressedFileExtensions` is `null`, `uncompressedFileExtensions` would be `null`?

I'm pretty sure that `Any()` will throw if `uncompressedFileExtensions` is null.

Reply:
We use `UncompressedFileExtensions?.Split () ?? new string[0]` which ensures that if UncompressedFileExtensions is null we always get an empty string array.

Reply:
Apparently I need to scroll right!

![image](https://user-images.githubusercontent.com/840039/41478290-bf4626ba-708c-11e8-9478-9b54bfc4d3d8.png)